src/app/modal/modal.html

```diff
@@ -123,7 +123,7 @@
                         <div class="clearfix"></div>
                     </div>
                     <!-- RESUME UPLOAD -->
-                    <div class="form-field upload-container" data-ngf-select data-ng-model="modal.ApplyService.form.resumeInfo" data-ngf-max-size="modal.configuration.maxUploadSize" data-ngf-multiple="false" data-ngf-accept="applyForm.$submitted=false; modal.validateResume($file);"  data-ng-if="!modal.hasAttemptedLIApply">
+                    <div class="form-field upload-container needsclick" data-ngf-select data-ng-model="modal.ApplyService.form.resumeInfo" data-ngf-max-size="modal.configuration.maxUploadSize" data-ngf-multiple="false" data-ngf-accept="applyForm.$submitted=false; modal.validateResume($file);"  data-ng-if="!modal.hasAttemptedLIApply">
```

Comment:
From the fastclick repo for the class.  
https://github.com/ftlabs/fastclick/blob/3db9f899c25b7b2e1517dc5cc17494ec9094bc43/README.md#ignore-certain-elements-with-needsclick